{
  "_from": "fastify-error@^0.1.0",
  "_id": "fastify-error@0.1.0",
  "_inBundle": false,
  "_integrity": "sha512-jyCEc3VPEc8/PUwzDQAM2JlXLK2BG6L19mMJzbGij0TfdY1sHF9pCnnAn6Vcoi84TMTBOJynNDQUMUz6cjRmBw==",
  "_location": "/fastify-error",
  "_phantomChildren": {},
  "_requested": {
    "type": "range",
    "registry": true,
    "raw": "fastify-error@^0.1.0",
    "name": "fastify-error",
    "escapedName": "fastify-error",
    "rawSpec": "^0.1.0",
    "saveSpec": null,
    "fetchSpec": "^0.1.0"
  },
  "_requiredBy": [
    "/fastify"
  ],
  "_resolved": "https://registry.npmjs.org/fastify-error/-/fastify-error-0.1.0.tgz",
  "_shasum": "7ca9d3708a2b1a74aefed0e5dde1b96d0390f718",
  "_spec": "fastify-error@^0.1.0",
  "_where": "/home/qodirov/Documents/samples/fastify-api/src/node_modules/fastify",
  "author": {
    "name": "Tomas Della Vedova"
  },
  "bugs": {
    "url": "https://github.com/fastify/fastify-error/issues"
  },
  "bundleDependencies": false,
  "deprecated": false,
  "description": "A small utility, used by Fastify itself, for generating consistent error objects across your codebase and plugins.",
  "devDependencies": {
    "ava": "^3.9.0",
    "standard": "^14.3.4",
    "tsd": "^0.11.0"
  },
  "homepage": "https://github.com/fastify/fastify-error#readme",
  "keywords": [
    "fastify",
    "error",
    "utility",
    "plugin"
  ],
  "license": "MIT",
  "main": "index.js",
  "name": "fastify-error",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/fastify/fastify-error.git"
  },
  "scripts": {
    "test": "standard && ava -v && tsd"
  },
  "types": "index.d.ts",
  "version": "0.1.0"
}
